mlrun/lists.py

```diff
@@ -179,10 +179,14 @@ def to_rows(self):
             "producer": ["producer", "spec.producer"],
             "sources": ["sources", "spec.sources"],
             "labels": ["labels", "metadata.labels"],
+            "uri": ["uri", "uri"],
```

Comment:
Please add a comment around this, saying that there's no `artifact.uri` field, and we fill it up later (actually, the only reason we need this row is so the name "uri" appears in the header row). Also, warn that the `uri` must be the last item here (otherwise the code later will not work).